packages/aws-cdk/lib/api/aws-auth/util.ts

```diff
@@ -13,7 +12,6 @@ export function readIfPossible(filename: string): string | undefined {
     }
     return fs.readFileSync(filename, { encoding: 'utf-8' });
   } catch (e: any) {
-    debug(e);
```

Comment:
I'm removing this debug. This only happens of the file exists but cannot be read, which seems super unliekly. The helper is called from only two places. If someone feels strongly about it, I can implement this in the call-sites.